packages/ingest/src/FtpProviderClient.ts

```diff
@@ -6,7 +6,11 @@ import Logger from '@cumulus/logger';
 import { Socket } from 'net';
 import { recursion } from './recursion';
 import { lookupMimeType, decrypt } from './util';
-import { FtpProviderClientListItem, ProviderClientListItem } from './types';
+import {
```

Comment:
Changes made to this class are entirely related to the parameterization of the .sync ProviderClient base class/this class and/or refactors related to param names/etc.